.rubocop.yml

```diff
@@ -149,7 +149,7 @@ Style/IfUnlessModifier:
   Description: Favor modifier if/unless usage when you have a single-line body.
   StyleGuide: https://github.com/bbatsov/ruby-style-guide#if-as-a-modifier
   Enabled: true
-  MaxLineLength: 100
+  MaxLineLength: 80
```

Comment:
It does apply to all code. This rule doesn't say that all conditionals that aren't one-liners have to fit within 80 characters. It says that if Rubocop thinks a conditional should be a one-liner, it should only apply if the one-liner will fit within 80 characters. If one-liners are more than 80 characters, that's OK. We just don't want Rubocop to force the use of super long one-liners. Does that make sense?